### DISCUSSION TITLE
Leistungsmessung und Ansteuerung einer Wärmepumpe

### DISCUSSION THREAD

**Discussion by sgrafcs** [camueller/SmartApplianceEnabler#344] [Installation und Betrieb]
Ich habe eine Wolf CHA 10 und messe deren Stromverbrauch mit einem 3P Zähler mit S0. Der S0 hängt am Pi auf welchem SAE 1.6.20 läuft. 
Solange ich nur die Leistung ermittle wird diese auch im Sunny Portal - Aktuelle und pro Tag - angezeigt.
Sobald ich ein Zeitplan für die Steuerung eines GPIOs eintrage wird die Leistungsaufnahme im Sunny-Portal nicht mehr angezeigt, obwohl die WP läuft.

Ob das Schalten klappt konnte ich bisher noch nicht prüfen, mir ist aber die Verbrauchsermittlung wichtiger.

Was habe ich falsch gemacht?

<img width="352" alt="image" src="https://user-images.githubusercontent.com/32098328/190858759-9ea4090c-d452-4a08-b3b8-1caa1749b7f5.png">

<img width="352" alt="image" src="https://user-images.githubusercontent.com/32098328/190858801-369a2ef6-79dc-4cf3-a86e-d71e1d905de7.png">

<img width="352" alt="image" src="https://user-images.githubusercontent.com/32098328/190858820-e6fdd22e-dad7-45c5-9c70-d40e775ae9c1.png">

<img width="352" alt="image" src="https://user-images.githubusercontent.com/32098328/190858876-7fa7ddef-2cc1-4ba9-ac9e-1cbf06e0126e.png">

**Comment by camueller** [comment]
Wenn Du einen Schalter konfiguriert hast, wir der Verbrauch im Sunny Portal nur berückcihtigt, wenn dieser auch eingeschaltet ist.
Siehe https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/QA_DE.md#semp2

Entweder Schalter und Zeitplan löschen, oder dafürt sorgen, dass er den richtien Status hat.

**Reply by sgrafcs** [reply]
Danken, ja das erklärt alles.

Ich wollte damit die PV-Anhebung steuern. Die Leistungsermittlung will ich aber immer sehen.

Ich hab das jetzt so eingerichtet, dass ich für das Schalten ein zweites Gerät angelegt haben. 
Das hat keinen Zähler, aber einen GPIO-Port und ein Zeitplan, vielleicht geht das ja so. Wenn man wieder Sonne da ist, werde ich es ja sehen.

**Comment by camueller** [comment]
Für die PV-Anhebung mit zwei Geräten ist der [Master/Slave-Zähler](https://github.com/camueller/SmartApplianceEnabler/blob/2.0/doc/MasterSlaveMeter_DE.md) wichtig, damit die Leistungsaufnahme korrekt zugeordnet wird.
Zumindest hebe ich damit die Puffer-Temperatur meiner WP durch Überschussstrom an. 

Diese Schalterart ist aber erst ab SAE 2.0 verfügbar und ich habe kürzlich Fehler in diesem Bereich gefixt. Ob alles funktioniert, kann ich erst sagen, wenn die Heizperiode beginnt.

**Reply by sgrafcs** [reply]
Danke für den Hinweis, das Update auf 2.x hatte ich auch schon ins Auge gefasst.
